manifests/init.pp

```diff
@@ -281,6 +284,7 @@
   Boolean $manage_service                         = true,
   Boolean $service_enable                         = true,
   Enum['running', 'stopped'] $service_ensure      = 'running',
+  Optional[Hash] $service_override                = undef,
```

Comment:
I'd simplify this to:
```suggestion
  Hash $service_override                = {},
```

You'll need to modify all other instances as well, but this simplifies things. Also, the type in the template itself is stricter than what you have here. I'd make this the same so puppet-strings can properly extract it.